app/frontend/src/screens/Operator/Operator.jsx

```diff
@@ -35,35 +68,74 @@ const AnimatedReason = posed(Container)({
 
 export const Operator = (props) => {
   let animationIndex = 0;
+  const controls = useAnimation();
+  const [drag, changeDrag] = useState('y');
+
+  const disableDrag = useCallback(() => changeDrag(false), [drag]);
+  const enableDrag = useCallback(() => changeDrag('y'), [drag]);
+
+  const onDragEnd = useCallback((event, info) => {
+    const shouldClose =
+      info.velocity.y > 20 || (info.velocity.y >= 0 && info.point.y > 200);
+    if (shouldClose) {
+      controls.start("hidden");
+    } else {
+      controls.start("visible");
+    }
+  });
 
   return (
     <PageContainer stretched>
-      <Missing>
-        <PoseGroup animateOnMount={true}>
-          {
-            users.filter(user => user.status === "not_here").map((user, index) => (
-              <AnimatedReason index={animationIndex++} key={index}>
-                <AvatarExpanded kind={user.avatar.kind} name={user.name} delay={animationIndex} details={user.reason} status={user.status} />
-              </AnimatedReason>
-            ))
-          }
-        </PoseGroup>
-      </Missing>
-      <TheRest row>
-        <PoseGroup animateOnMount={true}>
-          {
-            users.filter(user => user.status !== "not_here").sort((user1, user2) => {
-              if (user1.status === user2.status) return 0;
-              if (user1.status === "here" && user2.status === "not_answered") return 1;
-              if (user2.status === "here" && user1.status === "not_answered") return -1;
-            }).map((user, index) => (
-              <AnimatedReason index={animationIndex++} key={index}>
-                <AvatarDetails key={index} kind={user.avatar.kind} name={user.name} status={user.status} />
-              </AnimatedReason>
-            ))
-          }
-        </PoseGroup>
-      </TheRest>
+      <ContentContainer scrollable stretched>
+        <Missing>
+          <PoseGroup animateOnMount={true}>
+            {
+              users.filter(user => user.status === "not_here").map((user, index) => (
+                <AnimatedReason index={animationIndex++} key={index}>
+                  <AvatarExpanded kind={user.avatar.kind} name={user.name}
+                                  delay={animationIndex} details={user.reason}
+                                  status={user.status}/>
+                </AnimatedReason>
+              ))
+            }
+          </PoseGroup>
+        </Missing>
+        <TheRest row>
+          <PoseGroup animateOnMount={true}>
+            {
+              users.filter(user => user.status !== "not_here").sort((user1, user2) => {
+                if (user1.status === user2.status) return 0;
+                if (user1.status === "here" && user2.status === "not_answered") return 1;
```

Comment:
Maybe put "here" "not_here" "not_answered"  in a global variables? Or is this not how it's done in js?